src/algo/thermal-loop/l500-thermal-loop.h

```diff
@@ -0,0 +1,80 @@
+// License: Apache 2.0. See LICENSE file in root directory.
+// Copyright(c) 2020 Intel Corporation. All Rights Reserved.
+
+#pragma once
+
+#include <vector>
+#include "../../types.h"
+
+namespace librealsense {
+namespace algo {
+namespace thermal_loop {
+namespace l500 {
+
+using fx_fy = std::pair< double, double >;
+
+#pragma pack( push, 1 )
+
+class l500_thermal_loop
+{
+public:
+    struct rgb_thermal_calib_info
+    {
+        static const int table_id = 0x317;
+        static const int resolution = 29;
+
+        struct table_meta_data
+        {
+            float min_temp;
+            float max_temp;
+            float reference_temp;  // not used
+            float valid;           // not used
+        };
+
+        struct table_data
```

Comment:
This is data per a specific temperature? `temp_data`? `temp_params`?

Reply:
Done